src/nft/components/details/AssetDetails.tsx

```diff
@@ -196,7 +196,7 @@ const AssetView = ({
   asset,
   dominantColor,
 }: {
-  mediaType: 'image' | 'video' | 'audio'
+  mediaType: 'image' | 'video' | 'audio' | 'embed'
```

Comment:
nit: could use `MediaType` enum below for this

Reply:
@JFrankfurt Good call